src/row.js

```diff
@@ -590,6 +591,7 @@ Row.prototype.exists = function(gaxOptions, callback) {
 Row.prototype.filter = function(filter, config, callback) {
   var reqOpts = {
     tableName: this.table.id,
+    appProfileId: this.appProfileId,
```

Comment:
I think we will need to locate `appProfileId` from `this.bigtable.appProfileId`.